### PR TITLE
753

### LINKED ISSUE #753: Webmaker app should not use calc
Doesn't work in 4.2 :(

http://caniuse.com/#feat=calc

cc @gvn @flukeout just so you know

### PR DESCRIPTION
**Pull request by gvn**
R? @k88hudson 

Resolves #753

### CODE REVIEW

**Comment by k88hudson** [comment]
Add screen looks good but spinner is still messed up
screenshot_2015-01-21-17-19-42

File: views/make/index.less [hunk 1]
```diff
@@ -166,7 +166,7 @@
                 top: 5px;
                 left: 0;
                 width: 100%;
-                height: ~"calc(100% - 10px)";
+                height: 90%;
```
Comment (k88hudson) [inline views/make/index.less]: This is pretty weird spacing, i think what you want to do here is set 
```
left: 0;
right: 0;
top: 0;
bottom: 10px;
```

**Comment by k88hudson** [comment]
A couple of issues noted, mostly great. Thanks for cleaning this up!

**Comment by gvn** [comment]
R? @k88hudson

**Comment by k88hudson** [comment]
Yay!

**Pull request merged**